src/sentry/event_manager.py

```diff
@@ -875,6 +873,22 @@ def save(self, project_id, raw=False, assume_normalized=False):
 
         return event
 
+    def _get_event(self, project_id, event_id):
```

Comment:
nit: let's rename to get_event_from_storage, or load_event or something that specifies that we are fetching the event from the storage. Since we are in event manager, get event may mean that we are getting it from the input and what is critical is where we are getting it from.

Reply:
Done